labs/content/description.md

```diff
@@ -4,4 +4,4 @@ title: Description
 
 In this workshop we'll start from the very foundation of what makes React such a useful and transformational abstraction in modern website development. From this foundational perspective we'll explain the ins and outs of React ranging from JSX (why is this HTML in my JS!?), props, functional components, local component state, and everything in between to empower you to begin building incredible applications with React.
 
-You'll be coding a number of labs and building out real, representative React code to what many Fortune 500 companies are building. To prepare, simply bring a willingness to learn, a reasonably modern version of NodeJS and NPM, and an inquisitive nature and you'll leave with a great understanding and appreciation of React, the React ecosystem, and feel ready and able to build something great with React.
+You’ll be coding a number of labs and building out a real React application, similar to those being built by many Fortune 500 companies. To prepare, simply bring a willingness to learn, a reasonably modern version of NodeJS and NPM, and an inquisitive nature and you'll leave with a great understanding and appreciation of React, the React ecosystem, and feel ready and able to build something great with React.
```

Comment:
I think the single quote was right the first time. 😃

Reply:
Frickin Slack. It screws up quotes I’m copying/pasting